flocker/node/agents/test/test_blockdevice.py

```diff
@@ -460,6 +472,10 @@ def _incorrect_device_path_test(self, bad_value):
             volume.blockdevice_id, self.api.compute_instance_id(),
         )
 
+        # XXX There are N different cases to test here.  At least: filesystem
+        # exists with uuid, filesystem exists without uuid, filesystem does not
+        # exist.  Only testing does-not-exist case now.
```

Comment:
We usually require an issue, and then have a comment inline saying "We need to do X; see FLOC-1234".

In this case we expect to do that in next branch in the stack (Jean-Paul is working on it as we speak), so the XXX will be gone by end of day hopefully.